docs/admin/index.md

```diff
@@ -0,0 +1,150 @@
+# Perseo Context-Aware CEP Admin Guide
+
+## Deployment of Perseo
+
+Perseo can be built and installed directly from sources, but we strongly recommend deploying it using
+[Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/).
+
+If you need to install Docker, refer to the [Docker Installation](https://docs.docker.com/engine/installation/) web
+page. Then, checkout the docker-compose [install documentation](https://docs.docker.com/compose/install/) to install it
+if you don't already have.
+
+You can check if it works by running the following shell commands:
+
+```text
+# Docker
+docker --version
+
+# docker-compose
+docker-compose --version
+```
+
+Just in case, if you don't want Docker, take into account that Perseo CEP is a standard
+[Node.js](https://nodejs.org/es/) application and does not require more dependencies than the _Node.js interpreter_ and
+the [NPM](https://www.npmjs.com/) package utility.
+
+A [mongoDB 3.6](https://www.mongodb.com/) database should be working and accesible in order for Perseo to be started.
+
+### Using docker-compose
+
+If you want to quickly deploy all the Perseo CEP components to start experimenting ASAP, do the following:
+
+-   Download (or create locally) a copy of this
+    [docker-compose.yml](https://github.com/telefonicaid/perseo-fe/blob/master/docker-compose.yml) file.
+
+```yml
+version: '3.4'
+services:
+    perseo-mongo:
+        image: mongo:3.6
```

Comment:
Same comment as before.

Reply:
Fixed in https://github.com/telefonicaid/perseo-fe/pull/592/commits/fd872d8922203ee91da0747b762e37240c3a7973